internal/cmd/deploy/deploy.go

```diff
@@ -159,7 +161,12 @@ YAML:
 		spinner.Push("Creating change set")
 		changeSetName, createErr := cfn.CreateChangeSet(template, parameters, combinedTags, stackName, roleArn)
 		if createErr != nil {
-			panic(ui.Errorf(createErr, "error creating changeset"))
+			if createErr.Error() == noChangeFoundMsg {
+				fmt.Println(console.Green("\nChange set was created, but there is no change. Deploy was skipped."))
```

Comment:
If you put a `spinner.Pop()` just before this, you won't need the `\n` at the beginning of the message :)

Reply:
What a awesome utility!
https://github.com/aws-cloudformation/rain/blob/main/internal/console/spinner/spinner.go

I'll use this.